halt_system mismatch in arg count

7a45420 introduces build error when I run `ninja` in build

```
../kernel/exit.c: In function ‘halt_system’:
../kernel/exit.c:124:1: error: number of arguments doesn’t match prototype
 static void halt_system() {
 ^~~~~~
../kernel/exit.c:9:13: error: prototype declaration
 static void halt_system(int status);
             ^~~~~~~~~~~
```

`status` argument [was removed from the halt_system's definition](https://github.com/tbodt/ish/commit/7a45420ac0bd67852f2a898a6f0d565f67d8cbf9#diff-2fab5c2f0efa9b89d9b17cc88d9b37fbL116) but [the prototype still had that argument](https://github.com/tbodt/ish/blob/7a45420ac0bd67852f2a898a6f0d565f67d8cbf9/kernel/exit.c#L9).

Fix could be to remove the argument from the prototype or add it back but ignored?

--- a/kernel/exit.c
+++ b/kernel/exit.c
@@ -6,7 +6,7 @@
 #include "fs/fd.h"
 #include "fs/tty.h"
 
-static void halt_system();
+static void halt_system(void);
 
 static bool exit_tgroup(struct task *task) {
     struct tgroup *group = task->group;
@@ -121,7 +121,7 @@ noreturn void do_exit_group(int status) {
 }
 
 // always called from init process
-static void halt_system() {
+static void halt_system(void) {
     // brutally murder everything
     // which will leave everything in an inconsistent state. I will solve this problem later.
     for (int i = 2; i < MAX_PID; i++) {

--- a/kernel/exit.c
+++ b/kernel/exit.c
@@ -6,7 +6,7 @@
 #include "fs/fd.h"
 #include "fs/tty.h"
 
-static void halt_system(int status);
+static void halt_system();
 
 static bool exit_tgroup(struct task *task) {
     struct tgroup *group = task->group;
@@ -79,7 +79,7 @@ noreturn void do_exit(int status) {
         struct task *parent = leader->parent;
         if (parent == NULL) {
             // init died
-            halt_system(status);
+            halt_system();
         } else {
             leader->zombie = true;
             notify(&parent->group->child_exit);

